udfs/tests/dataform_testing_framework/deploy_and_run_tests.sh

```diff
@@ -246,6 +246,12 @@ main() {
       # which is mapped in dir_to_dataset_map.yaml
       local dataset_id
       dataset_id=$(sed -rn "s/${udf_dir}: (.*)/\1/p" <../../dir_to_dataset_map.yaml)
+      # Region suffixes are used to deploy UDFs globally to bqutil without naming conflicts
```

Comment:
```suggestion
      # Region suffixes are used to deploy UDFs globally to bqutil without naming conflicts
      # US multi-region datasets remain without suffix to avoid breaking changes to legacy users
```